src/SWP/Bundle/TemplatesSystemBundle/Service/ContainerService.php

```diff
@@ -14,138 +14,119 @@
 
 namespace SWP\Bundle\TemplatesSystemBundle\Service;
 
-use SWP\Bundle\TemplatesSystemBundle\Container\SimpleContainer;
-use SWP\Bundle\TemplatesSystemBundle\Model\Container;
-use SWP\Bundle\TemplatesSystemBundle\Widget\TemplatingWidgetHandler;
+use Doctrine\ORM\EntityManagerInterface;
+use SWP\Bundle\TemplatesSystemBundle\Factory\ContainerDataFactoryInterface;
+use SWP\Component\TemplatesSystem\Gimme\Model\ContainerDataInterface;
+use SWP\Component\TemplatesSystem\Gimme\Model\ContainerInterface;
 use SWP\Component\Common\Event\HttpCacheEvent;
-use Symfony\Bridge\Doctrine\RegistryInterface;
-use Symfony\Component\DependencyInjection\ContainerInterface;
 use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface as ServiceContainerInterface;
 
-class ContainerService
+/**
+ * Class RendererService.
+ */
+class ContainerService implements ContainerServiceInterface
 {
-    const OPEN_TAG_TEMPLATE = '<div id="swp_container_{{ id }}" class="swp_container {{ class }}" style="{% if height %}height: {{ height }}px;{% endif %}{% if width %}width: {{width}}px;{% endif %}{{styles}}"{% for value in data %} data-{{value.getKey()}}="{{value.getValue()}}"{% endfor %} >';
-    const CLOSE_TAG_TEMPLATE = '</div>';
-
+    /**
+     * @var ServiceContainerInterface
+     */
     protected $serviceContainer;
-    protected $objectManager;
-    protected $cacheDir;
-    protected $debug;
-    protected $renderer = false;
+
+    /**
+     * @var EntityManagerInterface
+     */
+    protected $entityManager;
```

Comment:
I think object manager (swp.object_manager.container) can be injected here instead